src/modules/assistants/builder/Builder.tsx

```diff
@@ -104,7 +103,7 @@ export function Builder({ thread, initialMessages }: Props) {
               size="lg"
               kind="tertiary"
               label="Back to home"
-              autoAlign
+              align="bottom-left"
```

Comment:
You can also keep `autoAlign`, it will make the button use Floating UI for the tooltip, it's marked experimental, but so far it was working well for me.

Reply:
By default popovers have z-index 6000, but the header is 8000 in Carbon (https://github.com/carbon-design-system/carbon/blob/main/packages/styles/scss/utilities/_z-index.scss#L19). That's why I positioned it bottom-left, so it's not covered with the header.